pages/api/arcade/showcase/projects/my.js

```diff
@@ -57,6 +58,7 @@ export default async function handler(req, res) {
       user: user.fields['Name'],
       color: p.fields['color'] || '',
       textColor: p.fields['textColor'] || '',
+      inRunning: false,
```

Comment:
I don't think you want this. we shouldn't be returning this value at all if the user hasn't voted

Reply:
removed!